### PR TITLE
Update and rename selectel.md

### PR DESCRIPTION
**Pull request by Irk-Grishin**
Selectel Storage API has changed from SWIFT to S3.

1) Updated links to the product.
2) Added a general description of the integration setup. 
3) Added a link to detailed instruction.
4) Changed the folder for this file

### CODE REVIEW

**Comment by dkocher** [comment]
Do you want to provide an updated connection profile [^1]?

[^1]: https://github.com/iterate-ch/profiles/blob/main/Selectel%20Cloud%20Storage.cyberduckprofile

File: protocols/s3/selectel.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Selectel S3 Storage
+====
+
+```{image} _images/selectel.png
+:alt: Selectel Drive Icon
+:width: 128px
+```
+
+> [Selectel Cloud Storage](https://selectel.ru/en/services/cloud/storage/) is a durable S3 for developers and businesses.
```
Comment (dkocher) [inline protocols/s3/selectel.md:9]: The page is not found.

**Review by dkocher — CHANGES_REQUESTED** [review]

**Review by Irk-Grishin — COMMENTED** [review]
I've done

File: protocols/s3/selectel.md (hunk 1 above)
Comment (Irk-Grishin) [reply protocols/s3/selectel.md]: I fixed it.

**Comment by Irk-Grishin** [comment]
@dkocher please tell me. What do I have to do to move this PR forward?

File: protocols/s3/selectel.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- In the Server field, enter the domain s3.ru-1.storage.selcloud.ru, where ru-1 is the pool that hosts the object storage.
+- In the Access Key ID field, insert the value of the Access key field from S3-key.
+- In the Secret access key field, insert the value of the Secret key field from the S3 key.
+- Press Connect.
+
+## Selectel S3 API Domains⁠
+
+Only authorized access is possible through S3 API domains. You have to use S3-key to work with containers and upload objects via:
+
+- ```s3.ru-1.storage.selcloud.ru/<container_name>``` domain with Path-Style addressing, used if Virtual-Hosted addressing is not enabled.
```
Comment (dkocher) [inline protocols/s3/selectel.md:39]: Single ticks should suffice.

File: protocols/s3/selectel.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Access can be configured by a user with role Account Owner or User Administrator in Selectel S3.
+
+Create a service user with a role with access to S3 object storage, see Role Model Access for details.
+Issue the user an S3 key.
+
+### Customize the client⁠
+
+- Install Cyberduck client.
+- Launch Cyberduck and click New Connection.
+- Select he S3 Selectel (or the Amazon S3 profile). To work with this profile, the container must have Virtual-Hosted addressing enabled, see the [Detailed instructions for connection](https://docs.selectel.ru/en/cloud/object-storage/tools/cyberduck/) for more details.
```
Comment (dkocher) [inline protocols/s3/selectel.md:29]: Typo `Select the…`

File: protocols/s3/selectel.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+```{image} _images/selectel.png
+:alt: Selectel Drive Icon
+:width: 128px
+```
+
+> [Selectel Cloud Storage](https://docs.selectel.ru/en/cloud/object-storage/) is a durable S3 for developers and businesses.
+
+## Connecting
+
+The integration Cyberduck with Selectel can be operated through a graphical or console client via the S3 API.
```
Comment (dkocher) [inline protocols/s3/selectel.md:13]: Not sure what you mean by `…can be operated through a graphical or console client via the S3 API.`.

File: protocols/s3/selectel.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+- Configure access.
+- Configure client.
+
+### Configure access
+
+Access can be configured by a user with role Account Owner or User Administrator in Selectel S3.
```
Comment (dkocher) [inline protocols/s3/selectel.md:20]: Remove line breaks.

File: protocols/s3/selectel.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+> [Selectel Cloud Storage](https://docs.selectel.ru/en/cloud/object-storage/) is a durable S3 for developers and businesses.
+
+## Connecting
+
+The integration Cyberduck with Selectel can be operated through a graphical or console client via the S3 API.
+
+- Configure access.
+- Configure client.
+
+### Configure access
```
Comment (dkocher) [inline protocols/s3/selectel.md:18]:
```suggestion
### Configure Access
```

File: protocols/s3/selectel.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+Access can be configured by a user with role Account Owner or User Administrator in Selectel S3.
+
+Create a service user with a role with access to S3 object storage, see Role Model Access for details.
+Issue the user an S3 key.
+
+### Customize the client⁠
```
Comment (dkocher) [inline protocols/s3/selectel.md:25]: The client is not customized. 

```suggestion
### Configuration
```

File: protocols/s3/selectel.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+- Install Cyberduck client.
+- Launch Cyberduck and click New Connection.
+- Select he S3 Selectel (or the Amazon S3 profile). To work with this profile, the container must have Virtual-Hosted addressing enabled, see the [Detailed instructions for connection](https://docs.selectel.ru/en/cloud/object-storage/tools/cyberduck/) for more details.
+- In the Server field, enter the domain s3.ru-1.storage.selcloud.ru, where ru-1 is the pool that hosts the object storage.
```
Comment (dkocher) [inline protocols/s3/selectel.md:30]:
```suggestion
- In the _Server_ field, enter the domain `s3.ru-1.storage.selcloud.ru`, where ru-1 is the pool that hosts the object storage.
```

File: protocols/s3/selectel.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+- In the Access Key ID field, insert the value of the Access key field from S3-key.
```
Comment (dkocher) [inline protocols/s3/selectel.md:31]:
```suggestion
- In the _Access Key ID_ field, insert the value of the Access key field from your S3 key.
```

File: protocols/s3/selectel.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+- In the Secret access key field, insert the value of the Secret key field from the S3 key.
```
Comment (dkocher) [inline protocols/s3/selectel.md:32]:
```suggestion
- In the _Secret Access Key_ field, insert the value of the Secret key field from your S3 key.
```

File: protocols/s3/selectel.md (hunk 3 above)
Comment (dkocher) [inline protocols/s3/selectel.md:29]: Add notice about optional installation of connection profile.

```
Connection profiles can be installed from *Preferences → Profiles*.
```

**Review by dkocher — CHANGES_REQUESTED** [review]

**Review by Irk-Grishin — COMMENTED** [review]
Let's move this MR forward

**Comment by dkocher** [comment]
Unfortunately there was a conflicting edit in 927721ebb82b5a71bdf50b4addc69f1e7f0dc787. Please rebase this pull request.

**Review by dkocher — APPROVED** [review]

**Review by dkocher — APPROVED** [review]

**Pull request merged**